src/backend/distributed/master/master_metadata_utility.c

```diff
@@ -62,6 +64,247 @@ static ShardPlacement * TupleToShardPlacement(TupleDesc tupleDesc,
 /* exports for SQL callable functions */
 PG_FUNCTION_INFO_V1(master_stage_shard_row);
 PG_FUNCTION_INFO_V1(master_stage_shard_placement_row);
+PG_FUNCTION_INFO_V1(citus_table_size);
+PG_FUNCTION_INFO_V1(citus_total_relation_size);
+PG_FUNCTION_INFO_V1(citus_relation_size);
+
+/*
+ * SupportFunctionForCitusSize is a helper function for all citus size functions.
```

Comment:
It would be good if you write these function comments in more detail, couple sentences about what this function does, how it does etc.

Also you do not need to add a empty comment line.